server/tests/server-test/tests/idlestate_test.cpp

```diff
@@ -72,11 +74,13 @@ TEST(IdleStateTest, possible_requests)
     EXPECT_EQ(player_1, game.currentPlayer().name());
     EXPECT_EQ(player_1, request_spy.last().at(0).toString());
     EXPECT_TRUE(containsRequest(request_spy.last().at(1).toJsonArray(), "end_turn"));
+    EXPECT_TRUE(containsRequest(request_spy.last().at(1).toJsonArray(), "construct_building"));
```

Comment:
I think, build should only be possible if you own a group of streets. Would this be possible?